handler/server_handler.go

```diff
@@ -0,0 +1,20 @@
+package handler
+
+import (
+	"dummy/database"
+	"dummy/repository"
+	"log"
+	"net/http"
+)
+
+func StartServer() {
+	var err error
+	repository.Repository.DB, err = database.SetupDatabase()
+
+	if err != nil {
+		panic(err)
+	}
```

Comment:
```suggestion
	repository.Repository.DB, err = database.SetupDatabase()
	if err != nil {
		panic(err)
	}
```
minor nit: usually, there's no space line between the operation and error-checking block.